### DISCUSSION TITLE
What can you add to the barrel in order to make it lighter?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#816] [Galaxy Trivia] [answered]
There is a barrel, filled to the top with water, which weighs 150 pounds.

**Comment by ParanoidUser** [comment]
@godkingjay / @samiam2013, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment]
You can add **holes** to the barrel so that the water will come out and make it lighter.

**Comment by SANKARAMDAS** [comment]
Make holes under the barrel, it will be lighter after that. 😅

**Comment by hasanerzi** [comment, marked as answer]
Add a water pump to barrel so you can pump the water out. Also you wont damage the barrel 🤣

**Comment by peterbaccaro** [comment]
Alternatively you could add **heat** to the barrel and slowly the water will evaporate and gradually make the barrel lighter.
